chart/sns-connector/README.md

```diff
@@ -0,0 +1,190 @@
+# OpenFaaS Pro SQS Connector
+
+The SNS connector can be used to invoke functions from an AWS SNS messages.
+
+See also: [Trigger functions from AWS SNS messages](https://docs.openfaas.com/openfaas-pro/sns-events/)
+
+## Prerequisites
+
+- Purchase a license
+
+  You will need an OpenFaaS License
+
+  Contact us to find out more [openfaas.com/pricing](https://www.openfaas.com/pricing)
+
+- Install OpenFaaS
+
+  You must have a working OpenFaaS installed.
+
+## Configure your secrets
+
+- Create the required secret with your OpenFaaS Pro license code:
+
+```bash
+$ kubectl create secret generic \
+    -n openfaas \
+    openfaas-license \
+    --from-file license=$HOME/.openfaas/LICENSE
+```
+
+- Create an AWS credentials secret:
+
+```bash
+$ kubectl create secret generic -n openfaas \
+  aws-sns-credentials --from-file aws-sns-credentials=$HOME/sns-credentials.txt
+```
+
+## Configure ingress
+
+To receive http calls from AWS SNS the callback url has to be publicly accessible.
```

Comment:
...

the below instructions show how to set up Ingress with a TLS certificate using Ingress Nginx. You can also use any other ingress-controller, inlets-pro or an Istio Gateway. Reach out to us if you need a hand.